vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/model/VpnDatabaseModels.kt

```diff
@@ -89,7 +89,7 @@ data class BucketizedVpnServiceStateStats(
     @Embedded val vpnServiceStateStats: VpnServiceStateStats
 )
 
-@Entity(tableName = "vpn_preferences")
+@Entity(tableName = "vpn_prefs")
```

Comment:
I have changed the name to make sure it's a new table. Because the `vpn_preferences` table is probably in production for some users